lms/djangoapps/courseware/views.py

```diff
@@ -412,9 +438,9 @@ def _index_bulk_op(request, course_key, chapter, section, position):
             return render_to_response('courseware/courseware.html', context)
         elif chapter is None:
             # Check first to see if we should instead redirect the user to an Entrance Exam
-            if settings.FEATURES.get('ENTRANCE_EXAMS', False) and course.entrance_exam_enabled:
-                exam_chapter, __ = get_entrance_exam_content_info(request, course)
-                if exam_chapter is not None:
+            if course_has_entrance_exam(course):
+                exam_chapter = get_entrance_exam_content(request, course)
+                if exam_chapter:
```

Comment:
Question: Is it possible that `course_has_entrance_exam(course)` is True but `exam_chapter` is undefined? This quadruple-nested `if` makes me uncomfortable (not that it is something changed by this PR).